packages/ndla-modal/src/Modal.js

```diff
@@ -491,6 +491,16 @@ class Modal extends React.Component {
     }
   }
 
+  componentWillReceiveProps(nextProps, prevProps) {
```

Comment:
Bruk ComponentDidUpdate i stedet. Tror kanskje vi kan fjerne isOpen som en prop også, så den ikke er kontrollert både utenfra og innenfra?

Reply:
Litt usikker på dette... Ser at det kanskje ikke er optimalt, men tror likevel det er beste løsningen.. greien er at i de fleste tilfeller ønsker man at Modal skal håndtere open/close selv. (Enklere å bruke i stateless components etc). Men i dette tilfellet må vi kunne kontrollere den utenfra.. Vi kunne vurdert å lage en wrapper for modal states håndtering til bruk i stateless components? Evt hvis du har et annet forslag på hvordan løse dette i Filestructure? Brukes forresten til å lukke modal utenfra når man adder en ny emnetilknytning for ressurs

Reply:
Fixed